src/main/java/com/thevoxelbox/voxelsniper/brush/type/stencil/StencilBrush.java

```diff
@@ -49,39 +51,60 @@
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
         SnipeMessenger messenger = snipe.createMessenger();
-        String firstParameter = parameters[1];
-        byte pasteParam;
+        String firstParameter = parameters[0];
+
         if (firstParameter.equalsIgnoreCase("info")) {
-            messenger.sendMessage(ChatColor.GOLD + "Stencil brush Parameters:");
-            messenger.sendMessage(ChatColor.AQUA + "/b schem [optional: 'full' 'fill' or 'replace', with fill as default] [name] -- Loads the specified schematic.  Allowed size of schematic is based on rank.  Full/fill/replace must come first.  Full = paste all blocks, fill = paste only into air blocks, replace = paste full blocks in only, but replace anything in their way.");
-            messenger.sendMessage(ChatColor.BLUE + "Size of the stencils you are allowed to paste depends on rank (member / lite, sniper, curator, admin)");
-            return;
-        } else if (firstParameter.equalsIgnoreCase("full")) {
-            this.pasteOption = 0;
-            pasteParam = 1;
-        } else if (firstParameter.equalsIgnoreCase("fill")) {
-            this.pasteOption = 1;
-            pasteParam = 1;
-        } else if (firstParameter.equalsIgnoreCase("replace")) {
-            this.pasteOption = 2;
-            pasteParam = 1;
+            messenger.sendMessage(ChatColor.DARK_AQUA + "Size of the stencils you are allowed to paste depends on rank (member / " +
+                    "lite, sniper, curator, admin)");
+            messenger.sendMessage(ChatColor.GOLD + "Stencil Brush Parameters:");
+            messenger.sendMessage(ChatColor.AQUA + "/b st (full|fill|replace) [n] -- Loads the specified stencil n. Allowed " +
```

Comment:
Is n supposed to be a number? If not change it.

Reply:
S like String is more likely appropriate.